### ISSUE TITLE
Add more clickable information at toolbar

### ISSUE THREAD

**Issue by zonky2** [contao/contao#6029]
### Description

It would be nice if more information about the current page were displayed in the toolbar and if these could be provided with a link to open them directly in the backend.

This would be, for example, the layout, template, page in the page tree or listing of the articles with link....

Discussed in the back-end session at the [Contao Camp 2023](https://github.com/contao-community-alliance/contao-camps/blob/main/2023/camp-2023.md#samstag-06052023).

**Comment by m-vo** [comment]
Just curious: What are the use cases for this?

**Comment by zonky2** [comment]
Basically, it should facilitate the mapping between frontend and backend. If you have pages that you do not know well, it is often difficult and time-consuming to get from the frontend to the editing point in the backend. The information is intended to make this easier.

During the discussion at the camp 23, it was also noted that the usual extensions for this use insert too much JavaScript/HTML dynamically, so that it is hardly possible to check the output code accordingly.

**Comment by k-webdesign** [comment]
Even if you know the page and have to do some typo corrections, this is pretty helpful. Also if the whole website is very extensive and you have to click a lot, it's much easier to have at least some shortcut/link to the page in the backend (page tree and articles).

**Comment by aschempp** [comment]
I fully agree. Would you mean to place this in the Symfony web inspector or the Contao preview toolbar? Some of the info is already present in the Symfony inspector, but that is only available to administrators 🤔

**Comment by zonky2** [comment]
I was referring to the Symfony toolbar - I had forgotten that it is only visible to admins (I am always an admin :D).

Then you would have to think about a solution, whether there is a separate "info toolbar" or whether you integrate it into the preview bar. But you usually activate it from the backend when you are on the corresponding page. Here, however, the information should be available, for example, if you don't know the location of the page...

**Comment by ausi** [comment]
You mean something like this?
![Bildschirmfoto 2023-05-11 um 10 59 17](https://github.com/contao/contao/assets/367169/4a9c9c42-5157-4587-b741-a9fc73a1d5da)
_(Screenshot from the RockSolid Frontend Helper extension)_

**Comment by zonky2** [comment]
yes, something like that...

But it would only need links to the backend to get to the page or article and no editing function in the FE.

The problem with the extension is that a lot of changes are made in the DOM, making it very difficult to check the actual source code.

**Comment by k-webdesign** [comment]
As zonky said, the extension is helpful but due to many DOM manipulations it's terrible to debug the website - and switching it on and off in the profile settings is cumbersome.

I think quickjumps like this will do it:
- edit settings of page ID XX
- edit articles of page ID XX
- publish/unpublish

**Comment by m-vo** [comment]
We could maybe even add a `data-content-element="<id>"` to the wrapper of each content element if in debug/preview mode. Then a piece of JS (brought in together with the toolbar ) could even add a similar overlay on a per-element level.

**Comment by aschempp** [comment]
> We could maybe even add a `data-content-element="<id>"` to the wrapper of each content element if in debug/preview mode. Then a piece of JS (brought in together with the toolbar ) could even add a similar overlay on a per-element level.

not the point of this issue though 😆

**Comment by fritzmg** [comment]
I quickly whipped something together:

<img src="https://github.com/contao/contao/assets/4970961/73106e94-e92f-4612-a3d2-473736ae43de" width="344">

Something like this?

**Comment by aschempp** [comment]
should this really be in the debug bar? I mean that would only be available to admins. The preview bar for example would be available to all users. But not sure that's currently the scope of the preview bar.

**Comment by fritzmg** [comment]
I suppose we could add these three buttons to the left of the preview bar may be:

* Edit page
* Edit content
* Edit layout

(or as a drop down or something)

**Comment by aschempp** [comment]
But wouldn't that follow-up with a million other requests that e.g. the rocksolid extension already provides? Like detecting a reader to edit that directly?
